server/views/reporting/reportingInstructionsForm.pug

```diff
@@ -20,18 +24,18 @@ block content
 
                         div.form-group
                             label.form-label(for="building") Building and street
-                            input#building.midMarginBottom.form-control(name="address1" type="text" value=data.buildingAndStreet1)
-                            input#street.form-control.block(name="address2" type="text" value=data.buildingAndStreet2)
+                            input#building.midMarginBottom.form-control(name="buildingAndStreet1" type="text" value=data.buildingAndStreet1)
+                            input#street.form-control.block(name="buildingAndStreet2" type="text" value=data.buildingAndStreet2)
 
                         div.form-group
                             label.form-label(for="town") Town or city
-                            input#town.form-control(name="address3" type="text" value=data.townOrCity)
+                            input#town.form-control(name="townOrCity" type="text" value=data.townOrCity)
 
                         div.pure-g
                             div.pure-u-1.pure-u-md-3-5
                                 div.form-group
                                     label.form-label(for="postcode") Postcode
-                                    input#postcode.form-control.small(name="postCode" type="text" value=data.postcode)
+                                    input#postcode.form-control.small(name="postcode" type="text" value=data.postcode)
                         div.pure-g
```

Comment:
In other places we've got postCode eg proposedAddress

Reply:
Yeah I didn't know which to go with. Royal mail seem to have it as one word